lightblue-latest/docker-compose.yml

```diff
@@ -13,3 +11,10 @@ mongodb:
   ports:
    - "28017:28017"
    - "27017:27017"
+applications:
+  image: lightblue/applications
+  command: /opt/jbossas7/bin/standalone.sh -b 0.0.0.0
```

Comment:
Container's localhost and host's localhost are not the same thing. Container's localhost is not available outside of the container. Since I don't know what IP address will be assigned to container's eth0, binding jboss to 0.0.0.0 seems like the easiest solution.